docs/content/TreeView.mdx

```diff
@@ -307,7 +307,12 @@ See [Storybook](https://primer.style/react/storybook?path=/story/components-tree
       </>
     }
   />
-  {/* <PropsTableSxRow /> */}
+  <PropsTableRow
+    name="count"
+    type="number"
+    description="Display a skeleton loading state with the specified count of items"
```

Comment:
"Display a skeleton loading state" makes me think that if the count is defined it will render a loading state. Maybe we should specify that the skeleton loading state is only rendered when the count is defined AND the state is "loading.".

```suggestion
    description="The number of items expected to be in the subtree. When in the loading state, the subtree will render a skeleton loading placeholder with the specified count of items"
```